src/bundle/Resources/config/services.yaml

```diff
@@ -1,2 +1,3 @@
 imports:
-    - { resource: services/**/*.yaml }
+    - { resource: services/papi.yaml }
+    - { resource: services/subscription_resolver.yaml }
```

Comment:
You don't need to change it, just place files in services dir - directly or nested ;)

Reply:
I wanted to avoid glob magic - it can bite you when loading some services conditionally.